packages/typography/stories/4-Ellipsis.stories.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { Ellipsis } from '@zendeskgarden/react-typography';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+export default {
+  title: 'Components/Typography/Ellipsis',
+  component: Ellipsis
+} as Meta;
+
+export const Default: Story = ({ title, tag }) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <Ellipsis title={title} tag={tag} style={{ width: 150, display: 'inline-block' }}>
+          Veggies es bonus vobis, proinde vos postulo essum magis.
+        </Ellipsis>
+      </Col>
+    </Row>
+  </Grid>
+);
+
+Default.argTypes = {
+  title: {
+    control: 'text'
```

Comment:
Is there a way to override the placeholder? If so, something like "Override the title attribute" would be helpful. Otherwise, it feels like this is controlling the Ellipsis children text.

Also, clearing the title control should restore the default `title`.

Reply:
I don't think there's a way to override the placeholder as it looks like it's [hard coded](https://github.com/storybookjs/storybook/blob/next/lib/components/src/controls/Text.tsx#L25). However, I've updated the Story to use the `Default.args.` The input is prepopulated with the title and when the input is cleared, the default `title` is now restored.